frontend/resourceadm/pages/MigrationPage/MigrationPage.tsx

```diff
@@ -246,5 +246,9 @@ export const MigrationPage = ({ navigateToPageWithError }: MigrationPageProps):
     );
   };
 
-  return <div className={classes.pageWrapper}>{displayContent()}</div>;
+  return (
```

Comment:
There are no tests for MigrationPage yet. These will be added once the final decisions are made on how this pag should work.